src/Parachain.sol

```diff
@@ -27,17 +32,22 @@ abstract contract Parachain {
         uint256 _amount
     ) internal {
         // Prepare remote call and send
-        uint64 transactRequiredWeightAtMost = 5000000000;
+
+        // The benchmarked of weight of an reportStakeDeposited dispatchable function on the corresponding pallet
```

Comment:
Can you use the rust name of the pallet functions please? Same for all the other report* functions in this file. Can you also remove "of" between "benchmarked of weight" to just read "benchmarked weight" for all similar comment please.